packages/train2go-bridge/test/parser.test.js

```
@@ -559,5 +559,60 @@ describe("parser", () => {
       expect(result.hrZones.swimming.z5).toEqual({ lower: 173, upper: 185 });
       expect(result.hrZones.swimming.z4Upper).toBe(172);
     });
+
+    it("should still emit hrZones.generic when the generic block is a sibling of hrzones-{id} (outside the per-user container, as in production T2G)", () => {
```

ESLint

[error] 'it' is not defined.

(no-undef)